requirements.txt

```diff
@@ -3,3 +3,4 @@ python-dateutil
 Flask
 PyYAML
 unicodecsv
+flask-cors
```

Comment:
We already have `flask-cors` in `requirements-dev.txt`.  We keep it optional given a pygeoapi deployment may have CORS enabled via Apache, for example.

Reply:
Sorry, that shouldn't have been committed.